### ISSUE TITLE
Cannot create Load Configuration

### ISSUE THREAD

**Issue by cmdstack** [lief-project/LIEF#1033]
**Describe the bug**
Need to create Load Configuration when does not exist.
Need to update existing Load Configuration.

**To Reproduce**
Load your PE first to `myPE`
```
>>> load_config = lief.PE.LoadConfiguration()
>>> myPE.load_configuration = load_config
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: property of 'Binary' object has no setter
```

**Expected behavior**
No error, update the PE.

**Environment (please complete the following information):**
NA

**Additional context**
NA

**NOTICE**

If the issue does not contain enough information to be reproduced,
it will be flagged as [incomplete](https://github.com/lief-project/LIEF/issues?q=is%3Aissue+is%3Aclosed+label%3AIncomplete)
and closed.

**/NOTICE**

**Comment by romainthomas** [comment]
Yes currently LIEF does not support adding or modifying PE's load config structure.
